seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,249 @@
+#-*- coding: utf-8 -*-
+import mock
+import __builtin__
+from copy import deepcopy
+from django.core.management.base import CommandError
+from seqr.models import Family, Individual, SavedVariant
+from pyliftover.liftover import LiftOver
+from seqr.views.utils.test_utils import ES_VARIANT1, ES_VARIANT2
+
+from django.core.management import call_command
+from django.test import TestCase
+
+PROJECT_NAME = u'1kg project n\u00e5me with uni\u00e7\u00f8de'
+PROJECT_GUID = 'R0001_1kg'
+ELASTICSEARCH_INDEX = 'test_index'
+INDEX_METADATA = {
+    "gencodeVersion": "25",
+    "hail_version": "0.2.24",
+    "genomeVersion": "38",
+    "sampleType": "WES",
+    "sourceFilePath": "test_index_alias_1_path.vcf.gz",
+}
+SAMPLE_IDS = ["NA19679", "NA19675_1", "NA19678", "HG00731", "HG00732", "HG00733"]
+
+liftover_to_38 = LiftOver('hg19', 'hg38')
+
+
+# To create a none return value for the variant at pos 1562437
+def mock_convert_coordinate(chrom, pos):
+    if pos == 1562437:
+        return
+    return liftover_to_38.convert_coordinate(chrom, pos)
+
+
+class LiftProjectToHg38Test(TestCase):
+    fixtures = ['users', '1kg_project']
+
+    @mock.patch.object(__builtin__, 'raw_input')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.logger')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_elasticsearch_index_samples')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_es_variants_for_variant_tuples')
+    def test_command(self, mock_get_es_variants, mock_get_es_samples, mock_logger, mock_input):
+        mock_get_es_samples.return_value = SAMPLE_IDS, INDEX_METADATA
+        es_variants = deepcopy([ES_VARIANT1, ES_VARIANT2])
+        es_variants[0]['xpos'] = 1046394160
+        es_variants[0]['ref'] = 'G'
+        es_variants[0]['alt'] = 'A'
+        es_variants[0]['familyGuids'] = ['F000001_1']
+        es_variants[1]['xpos'] = 1248203925
+        es_variants[1]['ref'] = 'TC'
+        es_variants[1]['alt'] = 'T'
```

Comment:
Why do you need to change the test fixture data? If its really necessary, do it at the top of the file and crate a constant, but ideally you should make the text work with the fixture or vice versa

Reply:
The data are the mock searching results from Elasticsearch. Therefore, they can't be in the fixture data.
I can put the updating to the top of the file as seqr/utils/elasticsearch/es_utils_tests.py does.

Reply:
Sorry, fixture here just means "fixed test data" not the fixture.json file. I meant why doesn't `ES_VARIANT1` work as is?